docs/ios/v2/how-to-guides/extend-capabilities/plugins/noise-cancellation.mdx

```diff
@@ -148,8 +153,7 @@ roomModel.isNoiseCancellationEnabled
 
 ## How to check if noise cancellation is enabled in the room
 
-To make noise cancellation work your room needs to have noise cancellation feature enabled. You can check if noise cancellation is enabled using roomModel.isNoiseCancellationAvailable. To enable noise canellation in your rooms, reach out to support@100ms.live or 100ms discord.
-
+To make noise cancellation work your room needs to have noise cancellation feature enabled. You can check if noise cancellation is enabled using roomModel.isNoiseCancellationAvailable. 
```

Comment:
To make noise cancellation work, your room needs to have noise cancellation feature enabled. You can check if it is enabled using `roomModel.isNoiseCancellationAvailable`.